mailer.js

```diff
@@ -96,6 +122,14 @@ module.exports = function (log) {
     return parts.join(', ')
   }
 
+  Mailer.prototype._constructLocationString = function (location) {
+    return constructLocationString(location)
+  }
+
+  Mailer.prototype._constructLocalTimeString = function (timestamp, locale) {
+    return constructLocalTimeString(timestamp, locale)
```

Comment:
Looks like it's called with `message.timeZone`, so the `timestamp` argument here is the odd one out.